app/code/Magento/UrlRewriteGraphQl/Model/Resolver/EntityUrl.php

```diff
@@ -75,7 +75,7 @@ public function resolve(
         if ($urlRewrite) {
             $result = [
                 'id' => $urlRewrite->getEntityId(),
-                'canonical_url' => $urlRewrite->getTargetPath(),
+                'relative_url' => $urlRewrite->getTargetPath(),
```

Comment:
Need to keep both of field and `canonical_url` need to mark as deprecated in the scheme
Thanks

Reply:
Added `canonical_url` field and marked as deprecated in the schema.